lightning/src/ln/features.rs

```diff
@@ -532,6 +532,14 @@ impl InvoiceFeatures {
 	}
 }
 
+impl Bolt12InvoiceFeatures {
+	/// Converts `Bolt12InvoiceFeatures` to `Features<C>`. Only known `Bolt12InvoiceFeatures` relevant
+	/// to context `C` are included in the result.
+	pub(crate) fn to_context<C: sealed::Context>(&self) -> Features<C> {
+		self.to_context_internal()
+	}
+}
+
```

Comment:
I don't think we can do this. It will work for MPP because we changed the spec to use the same feature bits. But more generally, any newly defined BOLT 12 invoice features may use bits that are defined differently in other contexts.

Reply:
Huh, why would we reuse feature bits in bolt 12 invoices? A link would be helpful if you have one. But, I can change the implementation to manually set mutual feature bits

Reply:
I could only find the comment where I brought it up:

https://github.com/lightning/bolts/pull/798#discussion_r978735091

But looks like we discussed it on Discord:

https://discord.com/channels/981260788004184144/981260788004184147/1019791999370539078

Haven't read through it again yet, so maybe I'm mistaken.

Reply:
Discussed offline, will keep it this way until/unless a feature bit conflict occurs